visallo/visallo
# Fix default state of showing only user visible properties (4.0.1)

Testing Instructions:
- Make sure sort search filter list doesn't show non user visible properties
- Make sure dashboard card configuration sort list doesn't show non user visible properties

Points of Regression: Sort properties list

CHANGELOG
Fixed: Sort on dashboard card showing non-user visible properties
